Makefile

```diff
@@ -170,7 +170,7 @@ img-rm-dang: ## remove dangling images (tagged None)
 
 
 pre-commit-all: ## run pre-commit hook on all files
-	@pre-commit run --all-files --hook-stage manual || (printf "\n\n\n" && git --no-pager diff --color=always)
```

Comment:
The main reason to revert this:
Because of `||` we don't fail the CI when we should.

Another reason:
Actually, all automatic pre-commit fixes will be pushed automatically be pre-commit.ci and the diff can be seen as a commit later.

When manually running pre-commit, I think it's not difficult to type `git diff` if you really want to see the diff.